tests/component/test_task.py

```diff
@@ -26,3 +42,103 @@ def test___tasks_with_different_names___hash___not_equal(generate_task):
     task2 = generate_task("MyTask2")
 
     assert hash(task1) != hash(task2)
+
+
+@pytest.mark.device_name("bridgeTester")
+@pytest.mark.grpc_xfail(
+    reason="Requires NI gRPC Device Server version 2.5 or later", raises=RpcError
+)
+@pytest.mark.parametrize(
+    "shunt_resistor_value, shunt_resistor_location, shunt_resistor_select, shunt_resistor_source, bridge_resistance, skip_unsupported_channels",
+    [
+        (100000, ShuntElementLocation.R1, ShuntCalSelect.A, ShuntCalSource.DEFAULT, 0.2, True),
+        (100000, ShuntElementLocation.R1, ShuntCalSelect.A, ShuntCalSource.DEFAULT, 0.2, False),
+        (100000, ShuntElementLocation.R3, ShuntCalSelect.A, ShuntCalSource.DEFAULT, 0.2, True),
+        (100000, ShuntElementLocation.R3, ShuntCalSelect.A, ShuntCalSource.DEFAULT, 0.2, False),
+    ],
+)
+def test___perform_bridge_shunt_cal___no_errors(
+    ai_bridge_task: nidaqmx.Task,
+    shunt_resistor_value,
+    shunt_resistor_location,
+    shunt_resistor_select,
+    shunt_resistor_source,
+    bridge_resistance,
+    skip_unsupported_channels,
+) -> None:
+    try:
+        ai_bridge_task.perform_bridge_shunt_cal(
+            ai_bridge_task.channels.name,
+            shunt_resistor_value,
+            shunt_resistor_location,
+            shunt_resistor_select,
+            shunt_resistor_source,
+            bridge_resistance,
+            skip_unsupported_channels,
+        )
+    except nidaqmx.DaqError as e:
+        if e.error_code != -201493:
+            raise
+
+
+@pytest.mark.device_name("bridgeTester")
+@pytest.mark.grpc_xfail(
+    reason="Requires NI gRPC Device Server version 2.5 or later", raises=RpcError
+)
+def test___perform_bridge_shunt_cal_default___no_errors(
+    ai_bridge_task: nidaqmx.Task,
+) -> None:
+    try:
+        ai_bridge_task.perform_bridge_shunt_cal(ai_bridge_task.channels.name)
+    except nidaqmx.DaqError as e:
+        if e.error_code != -201493:
```

Comment:
Add this import: `from nidaqmx.error_codes import DAQmxErrors`
And replace the hardcoded number with `DAQmxErrors.SHUNT_CAL_FAILED_OUT_OF_RANGE`